src/somef/__init__.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-__version__ = "0.9.6"
+__version__ = "0.9.7"
```

Comment:
Version is 0.9.6, can you please keep  it?
```suggestion
__version__ = "0.9.6"
```